Dockerfile.test.cpu

```diff
@@ -27,6 +27,13 @@ RUN apt-get install -y --no-install-recommends \
         build-essential \
         g++-4.8
 
+# install g++-7
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN apt-get update
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
```

Comment:
Can you add `-qq` to not spam the output?

Reply:
Yes.